### DISCUSSION TITLE
🖋: React NativeとExpoのアーキテクチャの違いを学習コンテンツで説明できないか？

### DISCUSSION THREAD

**Discussion by tani-takanori** [ws-4020/mobile-app-crib-notes#98] [Ideas]
アプリの起動方法で少し触れているが、実際のアプリ開発のときにexpoが利用できるかどうかという判断は開発効率に影響すると思っている。

頭に残るようにexpoのアーキテクチャや制限がわかるようなコンテンツがあると嬉しいのではないか？

**Comment by takei0w0** [comment]
#178 にて対応

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
